src/domain/provider-service.ts

```diff
@@ -65,11 +67,31 @@ export class ProviderService {
     }
   }
 
-  public async start (providerId: string, serviceType: string) {
-    const service = await this.tequilapiClient.serviceStart({
+  public async getFirstAccessPolicy (): Promise<AccessPolicyDTO | null> {
```

Comment:
It might be getter not to swallow error here and allow client to decide what to do in case of failure - `null` response doesn't tell much about why it happened, and `logger.info` might not be enough for the client.

Reply:
The consumer doesn't have to care or know what happened. We use the pattern of propagating errors, but all we do is report them to sentry which gets spammed and nobody really cares about it :D The less the app knows, the better:) null/obj is fully sufficient for the consumer.